docs-src/modules/ROOT/pages/loading-data.adoc

```diff
@@ -130,7 +130,10 @@ See xref:reference:expected-file-format[Expected File Format] for details.
 
 This loads the contents of the file to the Purchases table.
 
-The result of running `load()` is a `data_token_id``. The data token ID is a unique reference to the data currently stored in the system.
+The result of running `load` is a `data_token_id`. 
+The data token ID is a unique reference to the data currently stored in the system. 
+Data tokens repeatable queries: qeries performed against the same data token always run on the same input data
```

Comment:
This sentence reads weird to me in the context of the paragraph. Do we plan to make a section about repeatable queries later?

Reply:
Copy-pasta - this came from the queries docs, but I messed up a rewrite during the context switch. We could probably be more explicit about repeatable queries, but it is discussed some in the queries section.

Updated to hopefully read as actual english.